src/pricing.njk

```diff
@@ -62,7 +62,7 @@ description: Find out how much it costs to use FlowForge.
                             <note class="note text-teal-100">per Node-RED</br>instance/month</note>    
                         </div>
                     </div>
-                    <p class="mt-6 pb-2"><span class="inline font-normal">All of the FlowForge Premium features</span> included and <span class="inline font-normal">managed by us</span> to reduce your operational overheads.</p>
+                      <p class="mt-6 pb-2">Professionalise your Node-RED and FlowForge usage, increase collaboration, and <span class="inline font-normal">reduce operational overhead</span>.</p>
```

Comment:
Or Install FlowForge yourself or we will manage it for you.

Reply:
```suggestion
                      <p class="mt-6 pb-2">Install FlowForge yourself or we will <span class="inline font-normal">manage it for you</span>.</p>
```